examples/app_blink.rs

```diff
@@ -0,0 +1,33 @@
+#![feature(plugin, no_std, core, start)]
+#![no_std]
+#![plugin(macro_zinc)]
+
+extern crate core;
+extern crate zinc;
+
+use zinc::hal::timer::Timer;
+use zinc::hal::lpc17xx::{pin, timer};
+use zinc::hal::pin::GpioDirection;
+use zinc::hal::pin::Gpio;
+use core::option::Option::Some;
+
+#[zinc_main]
```

Comment:
@bharrisau `#[start]` boilerplate code removed and replaced with a simple syntax extension.